README.md

```diff
@@ -1,12 +1,12 @@
-OpenShift Application Platform
+Origin Application Platform
```

Comment:
Don't change - I think as the H1 we need to identify OpenShift first (community brand).  Origin is the name of the community technology.